doc/tasks/ecs.md

```diff
@@ -0,0 +1,76 @@
+#  Ecs - EasyCodingStandard
+
+Check coding standard in one or more directories.
+
+***Composer***
+
+```
+composer require --dev symplify/easycodingstandard
+```
+
+***Config***
+
+The task lives under the `ecs` namespace and has following configurable parameters:
+
+```yaml
+# grumphp.yml
+parameters:
+    tasks:
+        ecs:
+            config: ~
+            level: ~
+            whitelist_patterns: []
+            triggered_by: ['php']
+            fix: false
+            clear-cache: false
+            no-progress-bar: true
+
+```
+
+**config**
+
+*Default: null*
+
+If you want to use a different config file than the default easy-coding-standard.yml, you can specify your custom config file location with this option.
+
+
+**level**
+
+*Default: null*
+
+If you want to use a different level than the default one, specify it with this option.
+
+
+**whitelist_patterns**
+
+*Default: []*
+
+If you want to run on particular directories only, specify it with this option.
+
+**triggered_by**
+
+*Default: [php]*
+
+This option will specify which file extensions will trigger this task.
+
+
+**fix**
```

Comment:
Can you drop this feature?
It will introduce changes in your git status after you commit.
More info about auto-fixing: https://github.com/phpro/grumphp/blob/master/doc/faq.md#does-grumphp-support-automatic-fixing

Reply:
Of course, i did it @veewee  :)